bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo2Mu2Tau/SUSY_VBFH_HToAATo2Mu2Tau/VBFH_M125_ToAA_M10/SUSY_VBFH_M125_ToAA_M10_To2Mu2Tau_proc_card.dat

```diff
@@ -0,0 +1,5 @@
+import model NMSSMHET --modelname
+
+generate p p  > h01 j j $$ W+ W- Z, (h01 > a01 a01, a01 > tau+ tau-, a01 > mu+ mu-)
```

Comment:
Is this syntax good for VBF?

Reply:
i think this should be done with QCD suppression. P1_ccx_h01gg I already see this in subprocess directory which definitely does not look like vbf

Reply:
<img width="260" alt="image" src="https://github.com/cms-sw/genproductions/assets/17442656/846dee83-b663-4856-82f6-7704ff1249c2">

also you have things that are chimed in by other bsm particles... not sure if this is intended + how it's interpreted.

<img width="235" alt="image" src="https://github.com/cms-sw/genproductions/assets/17442656/ad0ab647-e111-4ba5-b426-23a82ee2cb26">

but definitely invaded by QCD...

Reply:
I missed that part, @Fengwangdong if you need pure EW process, please add "QCD==0"